AutomaticQC/SpikeClassifiers/imosSpikeClassifiersLimits.m

```diff
@@ -0,0 +1,119 @@
+function [limits] = imosSpikeClassifiersLimits(len)
+% function [limits] = imosSpikeClassifiersLimits(len)
+%
+% Load the types and limits of all imosTimeSeriesSpikeQC classifiers
+%
+% Inputs:
+%
+% len - the length of the time series (integer)
+%
+% Outputs:
+%
+% limits - A structure with the limits.
+%
+% Example:
+%
+% len = 100
+% [limits] = imosTimeSeriesSpikeQCLimits(len)
+% assert(limits.hampel.window.min==1)
+% assert(limits.hampel.window.max=len/2)
+% assert(limits.otsu_savgol.pdeg.min=2)
```

Comment:
this doesn't seem to match the structure below 
```
limits.hampel_half_window_width.min 
limits.otsu_savgol_pdeg.min
...
```

Reply:
yeap, another docstring wrong